--- planner-0.14.6/docs/libplanner/Makefile.am~	2010-10-14 21:41:27.000000000 +0200
+++ planner-0.14.6/docs/libplanner/Makefile.am	2016-05-05 18:52:53.472593724 +0200
@@ -165,8 +165,6 @@
 	    echo '-- Installing '$$i ; \
 	    $(INSTALL_DATA) $$i $(DESTDIR)$(TARGET_DIR); \
 	  done; \
-	  echo '-- Installing $(srcdir)/html/index.sgml' ; \
-	  $(INSTALL_DATA) $(srcdir)/html/index.sgml $(DESTDIR)$(TARGET_DIR); \
 	fi)
 
 uninstall-local:
