diff -Nur -x '*.orig' -x '*~' planner-0.14.3-2ubuntu1/libplanner/mrp-sql.c planner-0.14.3-2ubuntu1.new/libplanner/mrp-sql.c
--- planner-0.14.3-2ubuntu1/libplanner/mrp-sql.c	2008-06-24 14:12:55.000000000 +0000
+++ planner-0.14.3-2ubuntu1.new/libplanner/mrp-sql.c	2008-06-24 14:21:00.000000000 +0000
@@ -2406,7 +2406,7 @@
 
 #define WRITE_ERROR(e,c) \
 G_STMT_START \
-g_set_error(e,MRP_ERROR,MRP_ERROR_SAVE_WRITE_FAILED, sql_get_last_error (c)) \
+g_set_error(e,MRP_ERROR,MRP_ERROR_SAVE_WRITE_FAILED, sql_get_last_error (c)); \
 G_STMT_END
 
 
